runtime/shared_common/j9shr.tdf

```diff
@@ -2919,3 +2919,13 @@ TraceEvent=Trc_SHR_INIT_j9shr_init_Entry Overhead=1 Level=6 Template="INIT enter
 TraceEvent=Trc_SHR_INIT_j9shr_init_BootClassSharingEnabledByDefault Overhead=1 Level=1 Template="INIT j9shr_init: Bootstrap class sharing enabled by default"
 TraceException=Trc_SHR_INIT_j9shr_init_ExitOnNonFatal Overhead=1 Level=1 Template="INIT j9shr_init: exit on non-fatal error"
 TraceEvent=Trc_SHR_VMInitStages_Event_RunningInContainer Overhead=1 Level=1 Template="The JVM is running in container, class sharing is not enabled by default"
+
+TraceEntry=Trc_SHR_INIT_j9shr_storeGCHints_Entry Overhead=1 Level=2 Template="INIT::j9shr_storeGCHints() : Enter (hintsFlags=%zu heapSize1=%zu heapSize2=%zu forceReplace=%d)"
+TraceException=Trc_SHR_INIT_j9shr_storeGCHints_Null_Key Overhead=1 Level=2 Template="INIT::j9shr_storeGCHints() : Failed to generate the key for the GC hints"
+TraceException=Trc_SHR_INIT_j9shr_storeGCHints_Exit_Fail_To_Store  Overhead=1 Level=2 Template="INIT::j9shr_storeGCHints() : Failed to store the GC hints, returning NULL"
+TraceExit=Trc_SHR_INIT_j9shr_storeGCHints_Exit Overhead=1 Level=2 Template="INIT::j9shr_storeGCHints() : Exit - returnVal %p"
+TraceEntry=Trc_SHR_INIT_j9shr_findGCHints Overhead=1 Level=1 Template="INIT::j9shr_findGCHint() : Enter"
+TraceException=Trc_SHR_INIT_j9shr_findGCHints_Null_Key Overhead=1 Level=2 Template="INIT::j9shr_findGCHints() : Failed to generate the key for the GC hints"
```

Comment:
This should be level 1 because level 2 isn't shown before the startup phase change.

Reply:
Fixed